confgenerator/testdata/invalid/linux/logging-receiver_files_type_unsupported_parameter_transport_protocol/golden_error

```diff
@@ -1 +1,9 @@
-parameter "transport_protocol" in "files" type logging receiver "receiver_1" is not supported. Supported parameters: [include_paths, exclude_paths].
\ No newline at end of file
+the agent config file is not valid. detailed error: [8:7] unknown field "transport_protocol"
```

Comment:
We can remove some of these validation tests as redundant. Follow-up work.

Reply:
Which one(s) are redundant?

Reply:
A bunch of these tests have the form "invalid field X for type Y", because we had to explicitly check each field in the old code.

Now with the validate tags, I think we can just have a single such test.